core/scripts/server.js

```diff
@@ -125,10 +127,22 @@ app.post('/invalidate', invalidateCache)
 app.get('/invalidate', invalidateCache)
 
 app.get('*', (req, res, next) => {
+  if (NOT_ALLOWED_SSR_EXTENSIONS_REGEX.test(req.url)) {
+    utils.apiStatus(res, 'Vue Storefront: Resource is not found', 404)
```

Comment:
```suggestion
    apiStatus(res, 'Vue Storefront: Resource is not found', 404)
```

Reply:
Yes. sorry. had opened project which uses older release and it has  utils imported and uses utils.apiStatus ,  looks like i  just copied from the wrong window and did check